src/plugins/mer/merplugin.h

```diff
@@ -57,15 +57,6 @@ private slots:
 
 private:
     QMap<QString, Sfdk::VirtualMachine *> m_stopList;
-
-#ifdef WITH_TESTS
-    void verifyTargets(const QString &vm, QStringList expectedKits, QStringList expectedToolChains, QStringList expectedQtVersion);
-    void testMerSshOutputParsers_data();
```

Comment:
testMerSshOutputParsers() and corresponding _data() are still there in the .cpp files, thus failing the build.

Reply:
fixed, these two were removed by mistake, so added back.